webapp/app/containers/Widget/components/Workbench/ConfigSections/TableSection/index.tsx

```diff
@@ -453,6 +456,19 @@ export class TableSection extends React.PureComponent<ITableSectionProps, ITable
                   </Select>
                 </Col>}
             </Row>
+            {!withPaging ? null : <Row>
```

Comment:
use `&&` operator to simplify the statement

Reply:
already fixed